contributing-docs/testing/unit_tests.rst

```diff
@@ -864,11 +864,7 @@ Once you enter the container, you might run regular pytest commands. For example
 Running Tests using Breeze from the Host
 ........................................
 
-If you wish to only run tests and not to drop into the shell, apply the
-``tests`` command. You can add extra targets and pytest flags after the ``tests`` command. Note that
-often you want to run the tests with a clean/reset db, so usually you want to add ``--db-reset`` flag
-to breeze command. The Breeze image usually will have all the dependencies needed and it
-will ask you to rebuild the image if it is needed and some new dependencies should be installed.
+If you wish to only run tests and not to drop into the shell, apply the ``tests`` command. You can add extra targets and pytest flags after the ``tests`` command. Note that often you want to run the tests with a clean/reset db, so usually you want to add ``--db-reset`` flag to breeze command. The Breeze image usually will have all the dependencies needed and it will ask you to rebuild the image if it is needed and some new dependencies should be installed.
```

Comment:
The breaklines here are actually needed. We don't want long lines. The CI will probably warn us